api/src/websocket/controllers/rest.ts

```diff
@@ -0,0 +1,57 @@
+import type WebSocket from 'ws';
+import type { Server as httpServer } from 'http';
+import type { AuthenticationState, AuthMode, WebSocketClient } from '../types';
+import env from '../../env';
+import SocketController from './base';
+import emitter from '../../emitter';
+import { refreshAccountability } from '../authenticate';
+import { handleWebsocketException, WebSocketException } from '../exceptions';
+import logger from '../../logger';
+import { trimUpper } from '../utils/message';
+import { WebSocketMessage } from '../messages';
+
+export class WebsocketController extends SocketController {
+	constructor(httpServer: httpServer) {
+		super(httpServer, 'WS REST', String(env.WEBSOCKETS_REST_PATH), {
+			mode: String(env.WEBSOCKETS_REST_AUTH).toLowerCase() as AuthMode,
+			timeout: Number(env.WEBSOCKETS_REST_AUTH_TIMEOUT) * 10000,
```

Comment:
is this perhaps meant to be `1000` instead of `10000`? (similar to `WEBSOCKETS_GRAPHQL_AUTH_TIMEOUT`)